opencti-platform/opencti-graphql/src/migrations/1681825498443-group-initialization.js

```diff
@@ -0,0 +1,26 @@
+import { executionContext, SYSTEM_USER } from '../utils/access';
+import { logApp } from '../config/conf';
+import { findAll, groupEditField } from '../domain/group';
+
+export const up = async (next) => {
+  const context = executionContext('migration');
+  const groupsEdges = await findAll(context, SYSTEM_USER);
```

Comment:
You should fetch only the groups that matches your need (with undefined values) and not with findAll cause you might miss some items.

Reply:
I try to useto listAllEntities with a filter, but I can't set the filtering value to undefined...
![image](https://user-images.githubusercontent.com/75783086/233078031-aa91386d-b18e-41b1-8b11-6d624aeaef69.png)